src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/PolicyVersion.kt

```diff
@@ -0,0 +1,7 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util
```

Comment:
It'd be good if we could refer to this everywhere, but thats a bit of a stretch for this ticket!
In the meantime, could we move this to sit in: `uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies/Codes.kt`?